src/app/components/contact-button/contact-dialog.component.html

```diff
@@ -6,24 +6,43 @@
 <form class="form-dialog" autocomplete="off" layout-align="center center">
     <md-input-container class="user-container" layout="row" layout-align="left">
         <label>Name</label>
+        <md-tooltip md-direction="bottom" md-delay="400">
```

Comment:
Bottom is the default configuration for md-direction so you dont need `md-direction="bottom"` in this case